packages/playground/app/assets/js/playground.js

```diff
@@ -0,0 +1,232 @@
+document.addEventListener("DOMContentLoaded", function() {
+    toggleNav();
+    pageFocus();
+    if (typeof lunr !== 'undefined') {
+        initSearch();
+    }
+    toggleSettings();
+    setTheme();
+});
+
+function toggleSettings() {
+  const settingsButton = document.getElementById("settings-button"),
```

Comment:
if we do not run this js through babel, just remove all ES6 from the file. Currently I spotted usage of const/let but maybe there are other things to be changed